modules/nodes-nodes-audit-config-about.adoc

```diff
@@ -6,7 +6,7 @@
 [id="about-audit-log-profiles_{context}"]
 = About audit log policy profiles
 
-Audit log profiles define how to log requests that come to the OpenShift API server, the Kubernetes API server, and the OAuth API server.
+Audit log profiles define how to log requests that come to the OpenShift API server, Kubernetes API server, OpenShift OAuth API server, and OpenShift OAuth server.
```

Comment:
I tested: WriteRequestBodies and AllRequestBodies profiles don't change OpenShift OAuth server's auditing behavior from "Default", "None" profile disables OpenShift OAuth server's auditing. The wording "define how to log requests that come" in the following table may not be accurate for WriteRequestBodies and AllRequestBodies profiles for OpenShift OAuth server.

Reply:
Ah, OK, I see there is a "`.small`" note for it, then no issue about the wording.